mender/templates/workflows-worker-deploy.yaml

```diff
@@ -52,6 +52,16 @@ spec:
         args: ["worker", "--excluded-workflows", "generate_artifact"]
 {{- end }}
 
+        env:
+        {{- if .Values.global.auditlogs }}
+        - name: HAVE_AUDITLOGS
+          value: "1"
```

Comment:
Here I cannot use `Values.global.auditlogs` because it would be "true" while the service expects "1", correct?

Reply:
Any value is good, 1 or true, doesn't matter. Just use the variable :)